lib/Page.js

```diff
@@ -375,8 +375,10 @@ class Page extends EventEmitter {
       watcher.cancel();
       throw e;
     }
-    await result;
+    const error = await result;
```

Comment:
Just for convention can result be something like navigationResultPromise? It is weird to say error = result.

Reply:
Done.